src/cargo-kani/src/context.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::args::KaniArgs;
+use anyhow::{bail, Context, Result};
+use std::cell::RefCell;
+use std::path::PathBuf;
+
+/// Contains information about the execution environment and arguments that affect operations
+pub struct KaniContext {
```

Comment:
nit: I prefer the name Session like rustc uses.

Reply:
I like that, will change KaniContext to KaniSession.